### PR TITLE
Add dlna_dmr component

### PR DESCRIPTION
**Pull request by StevenLooman**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14749

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.dlna_dmr.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Example configuration.yaml entry
+media_player:
+  - platform: dlna_dmr
+    url: http://192.168.0.10:9197/dmr
+    name: "TV living room"
+```
+
+Please note that some devices, such as Samsung TVs, are rather picky about the source used to play from. The TTS service might not work in combination with these devices. If the play_media service does not work, please try playing from a DLNA/DMS (such as [MiniDLNA](https://sourceforge.net/projects/minidlna/)).
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/media_player.dlna_dmr.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/media_player.dlna_dmr.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The `dlna_dmr` platform allows you to control a [DLNA Digital Media Renderer](https://www.dlna.org/), such as DLNA enabled TVs or radios.
+
+To add a DLNA DMR device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: dlna_dmr
+    url: http://192.168.0.10:9197/dmr
+    name: "TV living room"
```
Comment (frenck) [inline source/_components/media_player.dlna_dmr.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good overall @StevenLooman!
Nevertheless, left you 2 comments, could you take a look? 👍

**Comment by StevenLooman** [comment]
Thank you for the review, will fix it soon!

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for the updates @StevenLooman!
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_components/media_player.dlna_dmr.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+description: "Instructions on how to integrate a DLNA DMR device into Home Assistant."
+date: 2018-06-01 14:27
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dlna.png
+ha_category: Media Player
+featured: false
+ha_release: 0.74
```
Comment (MartinHjelmare) [inline source/_components/media_player.dlna_dmr.markdown]: 0.76

File: source/_components/media_player.dlna_dmr.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: dlna_dmr
+    url: http://192.168.0.10:9197/dmr
+```
+
+Please note that some devices, such as Samsung TVs, are rather picky about the source used to play from. The TTS service might not work in combination with these devices. If the play_media service does not work, please try playing from a DLNA/DMS (such as [MiniDLNA](https://sourceforge.net/projects/minidlna/)).
+
+## {% linkable_title Configuration %}
```
Comment (MartinHjelmare) [inline source/_components/media_player.dlna_dmr.markdown]: Move this above the paragraph where we talk about the example config.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/media_player.dlna_dmr.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+To add a DLNA DMR device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: dlna_dmr
+    url: http://192.168.0.10:9197/dmr
+```
+
+## {% linkable_title Configuration %}
```
Comment (MartinHjelmare) [inline source/_components/media_player.dlna_dmr.markdown]: I meant that this header should be moved above the example paragraph.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by StevenLooman** [comment]
Great, thanks! Please be sure to tag me in any future issues when any problems turn up.

**Pull request merged**